Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine3.11
```

Comment:
Always get error due to the internet issue, need someone help to test the Dockerfile can be used

```
npm notice 
npm notice New minor version of npm available! 7.4.3 -> 7.5.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.5.0>
npm notice Run `npm install -g npm@7.5.0` to update!
npm notice 
npm ERR! code ECONNRESET
npm ERR! network aborted
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-01-30T04_14_25_447Z-debug.log
The command '/bin/sh -c npm install' returned a non-zero code: 1
```

Reply:
Are you good to go now, @Aisuko?

Reply:
Hard to say, but I always get stuck on those problems.

Reply:
Same here, Dockerfile doesn't build:

```
 ---> Running in 9bddba93ef3d
npm notice 
npm notice New patch version of npm available! 7.5.1 -> 7.5.2
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v7.5.2>
npm notice Run `npm install -g npm@7.5.2` to update!
npm notice 
npm ERR! code 1
npm ERR! path /app/node_modules/mozjpeg
npm ERR! command failed
npm ERR! command sh -c node lib/install.js
npm ERR! ⚠ spawn /app/node_modules/mozjpeg/vendor/cjpeg ENOENT
npm ERR!   ⚠ mozjpeg pre-build test failed
npm ERR!   ℹ compiling from source
npm ERR!   ✖ Error: Command failed: /bin/sh -c autoreconf -fiv
npm ERR! /bin/sh: autoreconf: not found
npm ERR! 
npm ERR! 
npm ERR!     at /app/node_modules/bin-build/node_modules/execa/index.js:231:11
npm ERR!     at runMicrotasks (<anonymous>)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:94:5)
npm ERR!     at async Promise.all (index 0)

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-06T11_23_00_158Z-debug.log

```

 neither `make setup-libs`:

```
npm ERR! Linux 4.15.0-135-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! code EMISSINGARG

npm ERR! typeerror Error: Missing required argument #1
npm ERR! typeerror     at andLogAndFinish (/usr/share/npm/lib/fetch-package-metadata.js:31:3)
npm ERR! typeerror     at fetchPackageMetadata (/usr/share/npm/lib/fetch-package-metadata.js:51:22)
npm ERR! typeerror     at resolveWithNewModule (/usr/share/npm/lib/install/deps.js:456:12)
npm ERR! typeerror     at /usr/share/npm/lib/install/deps.js:457:7
npm ERR! typeerror     at /usr/share/npm/node_modules/iferr/index.js:13:50
npm ERR! typeerror     at /usr/share/npm/lib/fetch-package-metadata.js:37:12
npm ERR! typeerror     at addRequestedAndFinish (/usr/share/npm/lib/fetch-package-metadata.js:82:5)
npm ERR! typeerror     at returnAndAddMetadata (/usr/share/npm/lib/fetch-package-metadata.js:117:7)
npm ERR! typeerror     at pickVersionFromRegistryDocument (/usr/share/npm/lib/fetch-package-metadata.js:134:20)
npm ERR! typeerror     at /usr/share/npm/node_modules/iferr/index.js:13:50
npm ERR! typeerror This is an error with npm itself. Please report this error at:
npm ERR! typeerror     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/michael/dev/layer5/npm-debug.log
Makefile:2: recipe for target 'setup-libs' failed
make: *** [setup-libs] Error 1
```
```

Reply:
Good, I will check the Dockerfile contents.

Reply:
using `nvm` fixed the errors I got

Reply:
Ok, I will try nvm